coderd/notifications/notifications_test.go

```diff
@@ -700,7 +699,9 @@ func TestNotificationTemplates_Golden(t *testing.T) {
 			name: "TemplateWorkspaceDeleted",
 			id:   notifications.TemplateWorkspaceDeleted,
 			payload: types.MessagePayload{
-				UserName: "Bobby",
+				UserName:     "Bobby",
```

Comment:
Note to self: Should I still be using a payload struct in these test cases? We do use all their fields in the test, but we don't use the payload struct itself.